### PR TITLE
preventing ITF failures causing a blocked sub

### PR DESCRIPTION
**Pull request by tblackwe**
## Summary
For Forms 0995 and 526EZ, before a Veteran can create a claim, we check the status of their `Intent to File` and create one on the Veterans behalf if it fails.

If those API requests fail or timeout, the Veteran is prevented from continuing. This PR removes that block, and updates the messaging to the Veteran to be more clear. 

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
https://github.com/department-of-veterans-affairs/va.gov-team/issues/82619

## Testing done
- _Describe what the old behavior was prior to the change_ 
  - If an error was triggered within ITF, the Veteran was not presented a `Continue` button.  
- _Describe the steps required to verify your changes are working as expected_
  - In local testing, authenticate and navigate to 526 or 995 flow, attempt to create claim, observe new info box and continue to claim
- _Describe the tests completed and the results_
  - manually tested 526 and 995 on both mobile and desktop, continuing past info box to application
  - updated unit tests

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |  526 image      |  526 image |
| | | image |
||995 image |995 image |
| Desktop |   526  image     |  526 image |
| | 0995 image | 0995 image |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Comment by tblackwe** [comment]
I reviewed the codeclimate issues. They are valid concerns but fixing them is outside the scope of this ticket.

File: src/applications/disability-benefits/all-claims/content/itfWrapper.jsx [hunk 1]
```diff
@@ -64,14 +64,42 @@ export const itfError = (
...
-        We’re sorry. Your Intent to File request didn’t go through because
-        something went wrong on our end. For help creating an Intent to File a
-        Claim for Compensation, please call Veterans Benefits Assistance at{' '}
-        <va-telephone contact={CONTACTS.VA_BENEFITS} />, Monday through Friday,
-        8:00 a.m. to 9:00 p.m. ET. Or, you can fill out VA Form 21-0966 and
-        submit it to:
+        You can continue to file your claim or call us to confirm.
       </p>
-      {claimsIntakeAddress}
+      <strong>What an intent to file means: </strong>
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/disability-benefits/all-claims/content/itfWrapper.jsx:69]: It is all lower case on the linked page, except for when it is describing the proper for title of `Intent to File a Claim for Compensation and/or Pension, or Survivors Pension and/or DIC`

All lower case is correct

[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/disability-benefits/all-claims/content/itfWrapper.jsx:69]: I see what you mean about 'mixed' the other messages use Intent to File. Will need to circle back with design to sign off on it.

File: src/applications/disability-benefits/all-claims/content/itfWrapper.jsx [hunk 2]
```diff
@@ -64,14 +64,42 @@ export const itfError = (
...
+      <p className="vads-u-font-size--base">
+        An intent to file sets a potential start date (or effective date) for
+        your benefits. You then have up to 1 year to complete and file your
+        claim. And you may be able to get retroactive payments (payments for the
+        time between when you submitted your intent to file and when we your
+        your claim).
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/disability-benefits/all-claims/content/itfWrapper.jsx:75]: fixed

File: src/applications/appeals/995/content/itfWrapper.jsx [hunk 3]
```diff
@@ -45,23 +45,45 @@ export const itfExpander = (
...
+        your claim).
+      </p>
+      <br />
+      <strong>What you can do next:</strong>
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            You can continue and submit your claim online today—or when you’re
+            ready. If we don’t have a intent to file on record for this claim
+            that’s that’s within the past year, we’ll use the date you submit
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/appeals/995/content/itfWrapper.jsx:67]: fixed ty.

File: src/applications/appeals/995/content/itfWrapper.jsx [hunk 4]
```diff
@@ -45,23 +45,45 @@ export const itfExpander = (
...
+        time between when you submitted your intent to file and when we your
+        your claim).
+      </p>
+      <br />
+      <strong>What you can do next:</strong>
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            You can continue and submit your claim online today—or when you’re
+            ready. If we don’t have a intent to file on record for this claim
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/appeals/995/content/itfWrapper.jsx:66]: fixed ty

File: src/applications/appeals/995/content/itfWrapper.jsx [hunk 5]
```diff
@@ -45,23 +45,45 @@ export const itfExpander = (
...
+      <p className="vads-u-font-size--base">
+        You can continue to file your claim or call us to confirm.
+      </p>
+      <strong>What an intent to file means: </strong>
+      <p className="vads-u-font-size--base">
+        An intent to file sets a potential start date (or effective date) for
+        your benefits. You then have up to 1 year to complete and file your
+        claim. And you may be able to get retroactive payments (payments for the
+        time between when you submitted your intent to file and when we your
+        your claim).
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/appeals/995/content/itfWrapper.jsx:58]: fixed ty

**Comment by Mottie** [comment]
Added Cypress e2e tests for Supplemental Claims & 526.

Note: I noticed minor differences in the ITF created alert title and the back navigation link vs button - I'm not sure if we need to be consistent.

File: src/applications/appeals/995/tests/995.cypress.helpers.js [hunk 6]
```diff
@@ -1,26 +1,97 @@
...
+  type = 'compensation',
+) => ({
   data: {
     id: '',
     type: 'evss_intent_to_file_intent_to_files_responses',
     attributes: {
       intentToFile: [
         {
-          id: '1',
+          id: {},
```
[the remark this answers is not in the document]
Comment (Mottie) [reply src/applications/appeals/995/tests/995.cypress.helpers.js]: Ahh, you're right. That should be a string. I think I copied that from somewhere, but now I don't remember 😬

**Review by anniebtran — COMMENTED** [review]
This looks good as far as the Supplemental Claims changes! There's a small copy change that was marked as resolved without the changes being made so I wanted to make sure we were okay with the language there. But otherwise LGTM

**Review by Mottie — APPROVED** [review]
LGTM

File: src/applications/appeals/995/content/itfWrapper.jsx [hunk 7]
```diff
@@ -45,23 +45,45 @@ export const itfExpander = (
...
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            You can continue and submit your claim online today—or when you’re
+            ready. If we don’t have a intent to file on record for this claim
+            that’s that’s within the past year, we’ll use the date you submit
+            your claim as a potential start date for your benefits.
+          </li>
+          <li>
+            Or you can call us confirm if you have an intent to file for this
```
[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/appeals/995/content/itfWrapper.jsx]: fixed ty

**Review by aurora-a-k-a-lightning — APPROVED** [review]

**Review by harshilmoradia — APPROVED** [review]

File: src/applications/appeals/995/content/itfWrapper.jsx [hunk 8]
```diff
@@ -45,23 +45,57 @@ export const itfExpander = (
...
+        </ul>
+      </p>
+      <br />
+      <strong>What you can do next:</strong>
+      <p className="vads-u-font-size--base">
+        <ul>
+          <li>
+            You can continue and submit your claim online today—or when you’re
+            ready. If we don’t have an intent to file on record for this claim
+            that’s within the past year, we’ll use the date you submit your
```
Comment (freeheeling) [inline src/applications/appeals/995/content/itfWrapper.jsx:79]: Looks correct here, but not in the included screenshot, where "that's" appears as "that's that's".

**Review by freeheeling — APPROVED** [review]
Noted a discrepancy between wording in screenshots and updated text content.

**Pull request merged**